docs/what-is-regression-games.mdx

```diff
@@ -2,58 +2,57 @@
+to supercharge your development process.
 
-![timeline feature](unused-content/session-insights/img/replay/timeline.png)
+Our vision is to build tools with the community that actually help you be more productive. There are four core principles driving the development of this framework:
+- **Ease of use** - Developers should be able to integrate and create new tests with minimal effort.
+- **Reliability** - In the dynamic world of games, reliability is key. Our framework is designed to handle the unexpected and give you the tools to create robust, dependable tests.
+- **Data driven** - Our framework collects comprehensive data, empowering you to build smarter automations and create more effective tests.
 
-## Availability Assurances
+Automated testing of games is challenging due to the complexity and dynamic behavior of games. This is why we
+provide a variety of approaches to create **bots** that can handle the unique constraints of your game. We aren't locked into either old school
+AI approaches or LLMs, but rather utilize any technique when it will actually result in a bot that can test your game reliably.
 
-The Regression Games team offers the following assurances for our initial testing with developers. Additionally, we have a dedicated Discord server for support, and can also create a Slack Connect channel, giving you direct access to our entire team. For access to
-these channels, send an e-mail to [aaron@regression.gg](mailto:aaron@regression.gg).
+## Highlighted Features
 
-* A single account can run at least 10 bots concurrently
-* The time from requesting a bot to be started to the time it connects to Unity will take less than 10 seconds if the bot is relying on dependencies that are common to most bots. This does not cover bots that may have lots of library dependencies, which may take longer.
+The table below highlights some of the main features of the framework.
 
-## Implemented Features
+| Feature | Description |
+| --- | --- |
+| [Plug-and-Play Setup](getting-started/installing-regression-games) | Simply add our SDK and add one GameObject to your scene to begin instantly collecting detailed information for tests and bots. Custom integration code is rarely needed to get started. |
+| [Get Started Quickly with Session Recording](getting-started/creating-your-first-automated-test) | Developers can create a new test simply by recording a play through of their scenario. |
+| [Computer Vision Capabilities](guides/interacting-with-ui-using-computer-vision) | In some cases, a bot needs to see the game to understand it. Our CV capabilities can extract information from game screenshots to help drive bot behavior and validate game state using approaches like OCR and object detection. |
+| [Exploratory Testing](guides/finding-game-breaking-bugs-with-stability-testing) | The framework collects a wide variety of data, used in both testing and bot building. |
```

Comment:
This title doesn't sound related to the description. Really, the description is a nothing-burger that basically says "yeah, we collect data" with no indication of what that data is or why the user should care

Reply:
Not sure how this got left over, I edited this to better reflect the title.